cardano-testnet/src/Testnet/Process/Cli/Transaction.hs

```diff
@@ -39,6 +42,11 @@ data TxBody
 
 data SignedTx
 
+data ReferenceScriptJSON
+
+data TxOutAddress = PKAddress PaymentKeyInfo
```

Comment:
`PubKeyAddress` would be better.

Reply:
Solved [here](https://github.com/IntersectMBO/cardano-node/commit/cf3dd298515061deb20d73196a11ab237340c8cd)